services/UserService.js

```diff
@@ -152,6 +129,17 @@ class UserService {
   hasAccessTo = (act, resource) => {
     return async (req, res, next) => {
       try {
+        if (req.user.isDisabled) {
```

Comment:
Yes. You got it right. A blocked user must be restricted from accessing any if the API endpoints, not just the `auth` routes. Instead of checking this condition in every controller, I thought it would be better just to write it in the midlleware passed to all the routes, `DRY` philosophy! 😆